lambdatimes/src/App.js

```diff
@@ -1,15 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
 import TopBar from './components/TopBar';
 import Header from './components/Header';
 import Content from './components/Content/Content';
+import styled from 'styled-components';
+
+const AppContainer = styled.div`
```

Comment:
I put styled components below so I can see my "logic" directly at the top. You could do that if you want